css_portfolio.css

```diff
@@ -243,3 +261,189 @@ body {
 		display: none;
 	}
 }
+
+.portfolio {
+	display: grid;
+	grid-template-columns: 1fr 1fr;
+	column-gap: 20px;
+	grid-template-rows: 1fr 2fr 2fr;
+	row-gap: 20px;
+	justify-items: center;
+	grid-template-areas: 
+		"h h"
+		"a b"
+		"c d";
+	height: max-content;
+	background-color: rgb(102, 110, 122);
+	padding: 50px;
+	width: 100%;
+	justify-content: center;
+}
+
+@media only screen and (max-width: 1200px) {
+	.portfolio {
+		grid-template-columns: 1fr;
+		grid-template-rows: 1fr 1fr 1fr 1fr 1fr;
+		grid-template-areas: 
+			"h"
+			"a"
+			"b"
+			"c"
+			"d";
+	}
+}
+
+@media only screen and (max-width: 820px) {
+	.projecttxt {
+		text-align: center;
+	}
+}
+
+.Portfolioh {
+	background-color: red;
+	grid-area: h;
+	text-align: center;
+	background-color: rgb(255, 255, 255);
+	width: max-content;
+	max-width: 500px;
+	height: max-content;
+	min-height: 170px;
+	padding: 0px 10px;
+	border-width: 2px;
+	border-style: solid;
+	border-radius: 10px;
+	outline-color: rgb(255, 255, 255);
+	outline-width: 2px;
+	outline-style: solid;
+	padding: 0px 25px;
+}
+
+.Portfolioh h2 {
+	font-weight: normal;
+	font-size: 2rem;
+}
+
+.Portfolioh p {
+	font-size: 1.2rem;
+}
+
+.projecta {
+	grid-area: a;
+	display: grid;
+	grid-template-columns: 1fr 1fr;
+	background-color: rgb(255, 255, 255);
+	border-width: 2px;
+	border-style: solid;
+	border-radius: 15px;
+	outline-color: rgb(255, 255, 255);
+	outline-width: 2px;
+	outline-style: solid;
+}
+
+@media only screen and (max-width: 820px) {
+	.projecta {
+		grid-template-columns: 1fr;
+		text-align: center;
+		height: max-content;
+		
+	}
+}
+
+.projectb {
+	grid-area: b;
+	display: grid;
+	grid-template-columns: 1fr 1fr;
+	background-color: rgb(255, 255, 255);
+	border-width: 2px;
+	border-style: solid;
+	border-radius: 15px;
+	outline-color: rgb(255, 255, 255);
+	outline-width: 2px;
+	outline-style: solid;
+}
+
+@media only screen and (max-width: 820px) {
+	.projectb {
+		grid-template-columns: 1fr;
+		text-align: center;
+		height: max-content;
+	}
+}
+
+.projectc {
+	grid-area: c;
+	display: grid;
+	grid-template-columns: 1fr 1fr;
+	background-color: rgb(255, 255, 255);
+	border-width: 2px;
+	border-style: solid;
+	border-radius: 15px;
+	outline-color: rgb(255, 255, 255);
+	outline-width: 2px;
+	outline-style: solid;
+}
+
+@media only screen and (max-width: 820px) {
+	.projectc {
+		grid-template-columns: 1fr;
+		text-align: center;
+		height: max-content;
+	}
+}
+
+.projectd {
```

Comment:
Do all of your projects have the same styles applied? If so, you could use the same class for all of them or alternatively, apply them in a group aka = .projecta, .projectb, .projectc, .projectd { }